### PR TITLE
Refactor resource placement group and documentation

### PR DESCRIPTION
**Pull request by Diptipowervs**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: ibm/service/power/ibm_pi_constants.go [hunk 1]
```diff
@@ -363,6 +365,12 @@ const (
 	PIVPNConnectionDeadPeerDetectionThreshold = "threshold"
 	PIVPNConnectionLocalGatewayAddress        = "local_gateway_address"
 	PIVPNConnectionVpnGatewayAddress          = "gateway_address"
+	PIVPNPolicyName                           = "policy_name"
+	PIVPNPolicyDhGroup                        = "policy_dh_group"
+	PIVPNPolicyEncryption                     = "policy_encryption"
+	PIVPNPolicyKeyLifetime                    = "key_lifetime"
+	PIVPNPolicyPFS                            = "policy_pfs"
+	PIVPNPolicyAuthentication                 = "policy_authentication"
```
Comment (michaelkad) [inline ibm/service/power/ibm_pi_constants.go:373]: Those have to me change to add prefix `Attr_` or `Arg_`

File: ibm/service/power/ibm_pi_constants.go [hunk 2]
```diff
@@ -329,6 +329,8 @@ const (
 
 	// Placement Group
 	PIPlacementGroupID      = "placement_group_id"
+	PIPlacementGroupName    = "group_name"
+	PIPlacementGroupPolicy  = "policy"
```
Comment (michaelkad) [inline ibm/service/power/ibm_pi_constants.go:333]: Those have to me change to add prefix `Attr_` or `Arg_`

**Review by michaelkad — CHANGES_REQUESTED** [review]

**Comment by Diptipowervs** [comment]
Closing PR due to multiple conflicts. New PR https://github.com/powervs-ibm/terraform-provider-ibm/pull/104

**Comment by Diptipowervs** [comment]
closed

**Pull request closed without merging**